tests/data/tag.spec.ts

```diff
@@ -73,6 +73,7 @@ describe('Tag', () => {
       const expectedSaveData = {
         basics: blankBasics,
         work: [],
+        education: [], // The order of the props matters! Why?
```

Comment:
It's because I've placed the education object after the work property when it was stringified. You can remove the comment.

Reply:
This may be a nice motivation to actually use the database within the browser rather than local storage.